server/monitor/src/main/resources/templates/overview.ftl

```diff
@@ -0,0 +1,90 @@
+<#--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+      <div><h3>${title}</h3></div>
+      <br>
+      <div class="center-block">
+        <table class="overview-table">
```

Comment:
It would be really nice if this page and others used [bootstrap's grid system](http://getbootstrap.com/css/#grid) (rather than `<table>` tags to structure the page).  If they did, the charts and tables would collapse into a single column as screen size of the browser shrinks.  You can test this with a desktop browser.  Check out the Accumulo website for an example.  The bootstrap code is very simple.  Something like below:

```html
<div class="container">
  <div class="row">
    <div class="col-md-6">Accumulo master table html</div>
    <div class="col-md-6">Zookeeper table html</div>
  </div> 
  <div class="row">
    <div class="col-md-6">Chart 1 html</div>
    <div class="col-md-6">Chart 2 html</div>
  </div> 
</div>
```